exporters/trace/jaeger/jaeger_test.go

```diff
@@ -823,49 +836,3 @@ func TestProcess(t *testing.T) {
 		})
 	}
 }
-
-func TestNewExporterPipelineWithOptions(t *testing.T) {
-	const (
-		serviceName     = "test-service"
-		eventCountLimit = 10
-		tagKey          = "key"
-		tagVal          = "val"
-	)
-
-	testCollector := &testCollectorEndpoint{}
-	tp, err := NewExportPipeline(
-		withTestCollectorEndpointInjected(testCollector),
-		WithSDKOptions(
-			sdktrace.WithResource(resource.NewWithAttributes(
-				semconv.ServiceNameKey.String(serviceName),
-				attribute.String(tagKey, tagVal),
-			)),
-			sdktrace.WithSpanLimits(sdktrace.SpanLimits{
```

Comment:
This test was removed because the only thing this tests that `TestExporterExportSpan` doesn't is that it validates the SDK event limits are correctly handled by the SDK. This is not related to the exporter and is only an configuration option of the SDK so it was removed here.